contrib/mod_tls.c

```diff
@@ -17775,27 +17788,22 @@ static int tls_ctx_set_next_protocol(SSL_CTX *ctx) {
 
 static int tls_ctx_set_options(SSL_CTX *ctx) {
 #if OPENSSL_VERSION_NUMBER > 0x009080cfL
+  SSL_CTX_clear_options(ctx, SSL_OP_CIPHER_SERVER_PREFERENCE);
```

Comment:
The `SSL_OP_CIPHER_SERVER_PREFERENCE` option is used here, although there is an if-guard at line 17804.
I think the if-guard should be applied here too

Reply:
Fair enough.  Done.